### ISSUE TITLE
is it possible to reject keyboard input from others

### ISSUE THREAD

**Issue by BisericaOrtodoxa** [FlavioFS/ParsecSoda#50]
One of my players is using a keyboard to controller remapper and is sending both keyboard and controller inputs. 

Is it possible to reject the keyboard inputs?

**Comment by SteveCraft1** [comment]
Wouldn't be easier to use a program that Block the keyboard input by itself or as a option?

**Comment by BisericaOrtodoxa** [comment]
> Wouldn't be easier to use a program that Block the keyboard input by itself or as a option?

Can't find one that lets windows see it but not parsec

**Comment by SteveCraft1** [comment]
Theres keyboard splinter,you can emulate the keyboard and block its inputs

**Comment by FlavioFS** [comment]
Yeah, I did a video on Keyboard Splitter some time ago. Awesome app Kofiz showed me. 🙂
[**Video - How to remap your keyboard into gamepads**](https://www.youtube.com/watch?v=XC_PI6n5JQk)

P.S.: Technically this is a client side issue, since the guest is sending multiple inputs.
